sherpa/astro/ui/utils.py

```diff
@@ -47,10 +47,39 @@
 info = logging.getLogger(__name__).info
 
 
-string_types = (str, )
+__all__ = ('Session',)
 
 
-__all__ = ('Session',)
+def get_pha_background(data, idval, bkg_id):
+    """Given a PHA data set, return the associated background.
+
+    Parameters
+    ----------
+    data : sherpa.astro.data.DataPHA instance
+    id : int or str
+        The identifier (only used for an error message).
+    bkg_id : int
+        The background identifier
+
+    Returns
+    -------
+    bkg : sherpa.astro.data.DataPHA instance
+
+    Raises
+    ------
+    IdentifierErr
+        If there is no bkg_id background component in data.
+
+    """
+
+    assert id is not None  # make sure id has been set
```

Comment:
Argh, this is a stupid line. It should either be removed or

```suggestion
    assert idval is not None  # make sure id has been set
```